ext/bg/js/options.js

```diff
@@ -205,6 +260,24 @@ function optionsVersion(options) {
             } else {
                 options.scanning.modifier = 'none';
             }
+        },
+        () => {
+            if (options.general.groupResults) {
+                options.general.resultOutputMode = 'group';
+            } else {
+                options.general.resultOutputMode = 'split';
+            }
+            options.general.compactTags = false;
```

Comment:
You don't need to set `compactTags` and `compcatGlossaries` defaults here since they are already specified in the default options object. The first thing that happens when options are loaded is that the default options are merged onto the user options, meaning any missing fields are filled in. The only time you have to manually do things in version blocks is if you want to base settings on a converted version settings used in a prior options version.

Reply:
Yeah, seems like they are useless at the moment. I wasn't sure if they should be set true by default for new users but false for existing users that haven't had the features in the first place.

Reply:
Yeah the idea is that you only have to write versions that transform prior data *or* you are fixing up broken options data that you introduced with a bug, which I have totally never had to do before.